File: openapi/code_samples/python/api_v1_organizations_{id}_settings/patch.py
settings = [
        {
            "name": "sso",
            "enabled": True
        },
        {
            "name": "dir_sync",
            "enabled": True
        }
    ]

scalekit_client.organization.update_organization_settings(
  organization_id='<organization_id>', settings=settings
)

File: openapi/code_samples/go/api_v1_organizations_{id}_settings/patch.go
settings := OrganizationSettings{

		Features: []Feature{

			{

				Name:    "sso",

				Enabled: true,

			},

			{

				Name:    "dir_sync",

				Enabled: true,

			},

		},

	}


organization,err := scalekitClient.Organization().UpdateOrganizationSettings(ctx, organizationId, settings)